### PR TITLE
Use the Google Elevation API to set elevation on all API calls

### PR DESCRIPTION
**Pull request by marumari**
## Description
Instead of setting the altitude to 0 for all API calls, let's attempt to at least be a little bit more realistic and use the actual elevation of the location in question.

## Motivation and Context
Although it doesn't solve an existing problem, it makes the API calls a bit more realistic with very little downside.

## How Has This Been Tested?
Ran it several times on my local machine, but it's a bit difficult to test.  I would like at least a bit of r+ on this.  The system is Ubuntu 16.04.1.

## Screenshots (if appropriate):
```
2016-08-08 12:13:06,582 [      MainThread][     utilities][    INFO] Location for '1234 34th Street, City, State, USA' found: 1234 34th Street, City, State, USA
2016-08-08 12:13:06,582 [      MainThread][     utilities][    INFO] Coordinates (lat/long/alt) for location: 43.743165 -90.21056 0.0
2016-08-08 12:13:06,656 [      MainThread][     runserver][    INFO] Parsed location is: 43.7431/-90.2106/349.6358 (lat/lng/alt)
2016-08-08 12:13:06,659 [      MainThread][        models][    INFO] Connecting to local SQLite database
2016-08-08 12:13:06,662 [   search_thread][        search][    INFO] Search overseer starting
2016-08-08 12:13:06,662 [   search_thread][        search][    INFO] Starting search worker threads
2016-08-08 12:13:06,663 [   search_thread][        search][    INFO] New location caught, moving search grid
```

Note that the (slightly confusing) coordinates INFO comes from pgoapi, so there's not much that can be done to suppress it.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

Note that this adds a new requirement, so make sure to run `pip install --upgrade -r requirements.txt`

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by huanga** [comment]
:+1: This is perfect, thank you!

File: runserver.py [hunk 1]
```diff
@@ -91,12 +92,22 @@
...
+        log.debug('Looking up coordinates in API')
         position = util.get_pos_by_name(args.location)
 
+    # Use the latitude and longitude to get the local altitude from Google
+    try:
+        url = 'https://maps.googleapis.com/maps/api/elevation/json?locations={},{}'.format(
+            str(position[0]), str(position[1]))
+        altitude = requests.get(url).json()[u'results'][0][u'elevation']
+        log.debug('Local altitude is: %sm', altitude)
+        position = (position[0], position[1], altitude)
```
Comment (invisiblek) [inline runserver.py]: Couldn't you do position[2] = altitude? No real need to reassign all the array values.

Comment (marumari) [inline runserver.py]: @invisiblek: No, tuples are immutable.

**Comment by Octylpy** [comment]
+1 looks good.

**Comment by chuyskywalker** [comment]
lgtm

**Pull request merged**